src/common/ITEMS/BFA/Others.js

```diff
@@ -0,0 +1,34 @@
+export default {
```

Comment:
I'd like to avoid an "others" file as much as possible for the BFA folders. How about naming this "Crafted"?